### DISCUSSION TITLE
一个特别诡异的问题，静态站点上传的文件和github actions的结果不同

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#7162] [Q&A]
我用github部署了一个Astro静态站点，然后用社区中的CLI传到halo上。但是我发现了一个特别奇怪的地方，就是我对文档进行了修改，重新部署之后，本地的内容甚至是github actihons的日志都能看到内容被修改了，但是上传到halo的文件却没有变动，我还是先把项目内容都清空的情况下再部署到halo的。所以这次上传的应该是新的内容，不知道为什么出现了这种现象。
下面是图片，就是我把index.mdx的一个链接加了个前缀路径/columns，可以看到本地和github actions的日志都表明生成的index.html的内容都有了改变，但是我打开halo静态站点的目录却发现还是以前生成的内容，但是这个更新时间是最新的，不清楚为什么会出现这种情况。
![image](https://github.com/user-attachments/assets/b6335218-0195-4b47-a8bc-55acb44c4d35)
![image](https://github.com/user-attachments/assets/bb1fbc2c-413d-42db-92ba-7b5e733f2045)
![image](https://github.com/user-attachments/assets/c46bafb6-fdc5-4c02-85da-c1b0d1790fa7)
![image](https://github.com/user-attachments/assets/8b8b2b48-3b9c-47e9-be79-40ed31d77f34)
![image](https://github.com/user-attachments/assets/b35006c3-c6cb-40df-a8a8-0e5aff5bdd0c)

**Comment by ruibaby** [comment]
检查缓存。

**Reply by IshuaWang** [reply]
应该是缓存问题，我看1pannel中static文件下的内容确实是最新的，但是halo打开的内容就不是新的，这个是哪里缓存造成的呢？浏览器的还是halo呢？可以避免这种现象发生吗？

**Reply by IshuaWang** [reply]
> 应该是缓存问题，我看1pannel中static文件下的内容确实是最新的，但是halo打开的内容就不是新的，这个是哪里缓存造成的呢？浏览器的还是halo呢？可以避免这种现象发生吗？

我仔细看了下，或许不是缓存的问题，我下载了halo静态站点这个插件里项目中的文件，发现和halo安装目录的static文件夹的文件并不相同，安装目录里的那个是最新的，但是halo控制台里的文件不是，但是两个更新时间是一样的，说明起码哪里有点问题，是不是这个插件对有些文件不会更新呢？
